src/libs/ReportActionsUtils.ts

```diff
@@ -415,29 +415,23 @@ function shouldReportActionBeVisibleAsLastAction(reportAction: OnyxEntry<ReportA
  * which includes a baseURL placeholder that's replaced in the client.
  */
 function replaceBaseURL(reportAction: ReportAction): ReportAction {
-    if (!reportAction) {
-        return reportAction;
-    }
+    const policyChangeActions = new Set<string>(Object.values(CONST.REPORT.ACTIONS.TYPE.POLICYCHANGELOG));
 
-    if (
-        !reportAction ||
-        (reportAction.actionName !== CONST.REPORT.ACTIONS.TYPE.POLICYCHANGELOG.INVITE_TO_ROOM && reportAction.actionName !== CONST.REPORT.ACTIONS.TYPE.POLICYCHANGELOG.REMOVE_FROM_ROOM)
-    ) {
-        return reportAction;
-    }
-    if (!reportAction.message) {
+    if (!reportAction?.message || !policyChangeActions.has(reportAction?.actionName)) {
```

Comment:
Confirming - won't this mean that the action _has to_ be in the `policyChangeActions` `Set`?

e.g. action has a message but action is not in that set... we want to return early?

Reply:
>For invite to room and remove from room policy change logs

Is the comment above still accurate?

Reply:
nvm I was reading this as "invite to room" and "remove from room policy changes logs" - probably because the method looks like it is generic for all report actions.

**nab suggestion**: rename it to `replaceBaseURLInPolicyChangeLogAction()`

Reply:
I'm not sure why we limited what report actions can and cannot replace the base URL. Would need @jasperhuangg's input.

The only catch I can think of is that if a user commented `"%baseUrl"` then it would also get caught by this, so thats a reason to limit this to only policy change actions. We don't expect normal report actions to need the baseURL for system messages right now.

Reply:
> Confirming - won't this mean that the action has to be in the policyChangeActions Set?

Correct!

Reply:
What would happen if we didn't restrict this to policy change actions:
https://github.com/Expensify/App/assets/17171674/4ebec343-c8cb-4b08-a57d-b357885e154d

Reply:
Yeah we def don't want that. Anyways LGTM!